erpnext_com/www/docs/user/manual/en/accounts/purchase-invoice.md

```diff
@@ -71,6 +71,12 @@ Sometimes you may need to hold an invoice from being submitted.
 
 ![PI Hold](/docs/assets/img/accounts/pi-hold.png)
 
+If you want to change the 'Release Date' after the Purchase Invoice is submitted, click on the **Hold Invoice** button on the top right.
+
+If you want to hold a submitted purchase invoice, click on **Create** then the **Block Invoice** option. If you want to unblock the invoice again then use 'Unblock Invoice' option.
+
+![Block PI](/docs/assets/img/accounts/pi_block.png)   
```

Comment:
@ShraddhaRanjane, please highlight the Block Invoice option in the screenshot with a sharp border RED rectangle.
take care that the rectangle does not leave any empty spaces around it

Reply:
Okay. I will do changes .